src/interpreter_funcs/voxelize.rs

```diff
@@ -203,7 +206,19 @@ impl Func for FuncVoxelize {
 
         let mut scalar_field = ScalarField::from_mesh(mesh, &voxel_dimensions, 0.0, growth_u32);
 
-        scalar_field.compute_distance_field(&(0.0..=0.0), FalloffFunction::Linear(1.0));
+        let volume_value_range = 0.0..=0.0;
+
+        scalar_field.compute_distance_field(&volume_value_range, 1.0, FalloffFunction::Linear);
+
+        if fill {
+            let mut scalar_field_inside =
+                scalar_field.extract_voxels_inside_volumes(&volume_value_range);
+            scalar_field_inside.apply_unary_voxel_function(UnaryVoxelFunction::SetConstant(0.0));
+            scalar_field.apply_binary_voxel_function(
+                &scalar_field_inside,
+                BinaryVoxelFunction::ReplaceIfValue,
+            );
+        }
 
         let meshing_range = if fill {
```

Comment:
This `if` shouldn't be necessary anymore as far as I understand and the meshing range can just be `(Bound::Included(-growth_f32), Bound::Included(growth_f32))`